cmd/spire-agent/cli/run/run.go

```diff
@@ -124,50 +113,52 @@ func (*RunCLI) Synopsis() string {
 	return "Runs the agent"
 }
 
-func parseFile(filePath string) (*runConfig, error) {
-	c := &runConfig{}
+func parseFile(path string) (*config, error) {
+	c := &config{}
+
+	if path == "" {
+		path = defaultConfigPath
+	}
 
 	// Return a friendly error if the file is missing
-	if _, err := os.Stat(filePath); os.IsNotExist(err) {
-		msg := "could not find config file %s: please use the -config flag"
-		p, err := filepath.Abs(filePath)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
instead of doing a separate stat, we should just check the error returned by ioutil.ReadFile

Reply:
I updated both call sites to use the ReadFile error instead.